polars/polars-core/src/frame/mod.rs

```diff
@@ -54,6 +54,7 @@ pub enum NullStrategy {
 pub enum UniqueKeepStrategy {
     First,
     Last,
+    KeepNone,
```

Comment:
Can we name this `None`? It is similar to `first`, `last`.

Reply:
I think `None` is a reserved keyword in Rust, right? I'll try, see what happens.

Reply:
I don't think so. Option is also an `Enum`. It's disciminant is only imported by default.

https://doc.rust-lang.org/stable/std/option/enum.Option.html

Reply:
Indeed, this works just fine!

Reply:
Great! Thanks for the PR!